internal/config/config_test.go

```diff
@@ -343,19 +373,7 @@ var _ = Describe("Config", func() {
 			})
 
 			It("is set and the value is the version", func() {
-				Expect(cfg.MinVersionRequired).To(Equal("1.3.0"))
-			})
-		})
-	})
-
-	Describe("invalid combinations", func() {
-		When("-dry-run and -parallel are specified together", func() {
-			BeforeEach(func() {
-				fakeArgs = append(fakeArgs, "-dry-run", "-parallel", "10")
-			})
-
-			It("succeeds", func() {
-				Expect(cfgErr).NotTo(HaveOccurred())
+				Expect(cfg.MinVersion).To(Equal(version.Must(version.NewVersion("1.3.0"))))
```

Comment:
praise: thanks to the change made in the validation function we can use this type of comparison